### PR TITLE
Editor: Use fflate in Loader.js.

### PR DESCRIPTION
**Pull request by Mugen87**
Related issue: #20941

**Description**

Replaces JSZip with fflate in `Loader.js`.

### CODE REVIEW

**Comment by Mugen87** [comment]
https://raw.githack.com/Mugen87/three.js/dev2/editor/index.html

File: editor/js/Loader.js [hunk 1]
```diff
@@ -697,35 +697,37 @@ function Loader( editor ) {
...
-			var object = new OBJLoader().setMaterials( materials ).parse( zip.file( 'model.obj' ).asText() );
+			var materials = new MTLLoader().parse( strFromU8( zip[ 'materials.mtl' ] ) );
+			var object = new OBJLoader().setMaterials( materials ).parse( strFromU8( zip[ 'model.obj' ] ) );
 			editor.execute( new AddObjectCommand( editor, object ) );
 
 		}
 
 		//
 
-		zip.filter( async function ( path, file ) {
```
Comment (Mugen87) [inline editor/js/Loader.js:717]: Was there a specific reason for using `filter()`? It seems it is just necessary to iterate through the object 🤔 .

Comment (mrdoob) [reply editor/js/Loader.js:717]: I don't remember 🤔

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**